### DISCUSSION TITLE
How do you import a Risor file into another Risor file ?

### DISCUSSION THREAD

**Discussion by serge-hulne** [deepnoodle-ai/risor#58] [Q&A] [answered]
How do you import a Risor file into another Risor file ?

**Comment by myzie** [comment]
Hi @serge-hulne - I'm enabling this in the Risor CLI now. See this PR:
https://github.com/risor-io/risor/pull/59

**Comment by serge-hulne** [comment]
Thank you!

**Comment by myzie** [comment, marked as answer]
Ok, this is available in Risor `v0.8.0`, which is on homebrew as well.

For example, add to a lib.risor file:

```go
func add(x, y) { x + y }
```

Then you can run the CLI with `risor -c "import lib; lib.add(3, 4)"`
